shraga_common/app/routers/run_flow.py

```diff
@@ -67,7 +67,7 @@ async def run_flow(
         if hasattr(req_body, "chat_id"):
             rsp.chat_id = req_body.chat_id
 
-        if keep and rsp.stats:
```

Comment:
This fix doesn't make sense to me 
if there's no stats, the underlying code won't do anything?